gateway-service-health/src/test/java/org/apache/knox/gateway/service/health/PingResourceTest.java

```diff
@@ -24,6 +24,6 @@ public class PingResourceTest {
   @Test
   public void testPingCdoGetontent() {
     PingResource pr = new PingResource();
-    Assert.assertEquals(pr.getPingContent(), pr.CONTENT);
+    Assert.assertEquals(PingResource.OK, pr.OK);
```

Comment:
What is this testing now? Just comparing constant values?

Reply:
`Refactor / Inline` removed the original, I put it back.